src/Client/Articles.php

```diff
@@ -117,7 +117,9 @@ public function slice(int $offset, int $length = null) : Sequence
                 $articles = [];
 
                 foreach ($result['items'] as $article) {
-                    if (isset($this->articles[$article['id']])) {
+                    if (isset($article['-invalid'])) {
+                        $articles[] = null;
```

Comment:
skipping it altogether ripples through other code which expects a number of elements in the array that matches the `count()`, breaking it